### DISCUSSION TITLE
How to make the node labels go inside the empty boxes in the Programming Workflow nodes?

### DISCUSSION THREAD

**Discussion by HariSekhon** [mingrammer/diagrams#882] [General]
In the Programming Workflow nodes the boxes are empty and the labels go underneath them instead of inside them as you would naturally draw.

Is it possible to put the labels inside the nodes?

### Example

```python
from diagrams import Diagram, Edge
from diagrams.programming.flowchart import Action
 
 with Diagram('Release Straight to Production!', direction='BT'):
    prod = Action("Prod")
    for _ in range(1, 4, 1):
        Action("Dev") >> Edge(label=f"Feature {_}") >> prod
```

![release_straight_to_production!](https://user-images.githubusercontent.com/2211051/232348771-e0def793-7e8e-4f03-8070-0b08ea1f85f4.png)

I've done a couple other diagrams and played with all the examples and they don't come out anywhere near as ugly as this (this is the first time I'm trying to do a workflow with the programming node set):

https://github.com/HariSekhon/Diagrams-as-Code

**Comment by benbjamin24** [comment]
Check out the c4 module. 
https://diagrams.mingrammer.com/docs/nodes/c4

**Comment by HariSekhon** [comment]
I ended up doing this in d2 instead:

https://github.com/HariSekhon/Diagrams-as-Code#devs-test-in-production

![](https://raw.githubusercontent.com/HariSekhon/Diagrams-as-Code/master/images/test_in_production.svg)
